fluentbit/conf/conf.go

```diff
@@ -255,8 +255,8 @@ const (
     # https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
     Name              stackdriver
     resource          gce_instance
-    stackdriver_agent {{.UserAgent}}
-    workers           {{.Workers}}
+    stackdriver_agent {{.UserAgent}}{{if .Workers}}
```

Comment:
(Optional) You could put it on a new line and do something like

    {{- if .Workers}}

to clean up the whitespace.

Reply:
Ah good idea. Changed.